index.js

```diff
@@ -0,0 +1,14 @@
+const validatePassword = (valid) => {
```

Comment:
Naming this `valid` is weird, it might not be valid. It is always a password though, that'd be better naming.